test/suites/secondary_redis.bash

```diff
@@ -0,0 +1,95 @@
+SUITE_secondary_redis_PROBE() {
+     if [ "`which redis-server`" = "" ]; then
+        echo "no redis-server"
+        return
+    fi
+    if [ "`which redis-cli`" = "" ]; then
```

Comment:
Please use the shell-builtin `[[` instead of `[`. Maybe run [shellcheck](https://github.com/koalaman/shellcheck) on this file.